ArmJoints joints
string routine
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose
---
ArmTrajectory arm_trajectory
